tests/epyccel/test_epyccel_functions.py

```diff
@@ -154,6 +154,35 @@ def f8(x,b):
     assert f(3,False) == f8(3,False)
     # ...
 
+
+def test_arguments_f9():
+    @types('int64[:]')
+    def f9(x):
+        x += 1
+
+    f = epyccel(f9)
+
+    x = np.zeros(10, dtype='int64')
+    x_expected = x.copy()
+
+    f9(x)
+    f(x_expected)
+    assert np.allclose(x, x_expected, rtol=1e-15, atol=1e-15)
```

Comment:
Shouldn't we check for exact equality here? This is an integer array...

Reply:
done